x-pack/test/functional/services/ml/data_visualizer_file_based.ts

```diff
@@ -54,6 +54,13 @@ export function MachineLearningDataVisualizerFileBasedProvider(
       await testSubjects.existOrFail('mlFileDataVisFileStatsPanel');
     },
 
+    async assertNumberOfFieldCards(number: number) {
+      const cards = await testSubjects.findAll('mlPageFileDataVisFieldDataCard');
+      if (cards.length !== number) {
+        throw new Error(`expected ${number} field cards to exist, but found ${cards.length}`);
```

Comment:
While this works, I'd suggest to use `expect` which is used for most of our assertions, something like

```
expect(cards.length).to.eql(number, `expected ${number} field cards to exist, but found ${cards.length}`);
```

Reply:
changed in https://github.com/elastic/kibana/pull/77251/commits/088c544a4f13fd43896f29fab3cac0afec715bec